Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel.swift

```diff
@@ -186,13 +193,20 @@ class FaceLivenessDetectionViewModel: ObservableObject {
 
     func initializeLivenessStream() {
         do {
-            guard let livenessSession = livenessService as? FaceLivenessSession else {
-                throw FaceLivenessDetectionError.unknown
+            if (abs(Self.attemptIdTimeStamp.timeIntervalSinceNow) > defaultAttemptCountResetInterval) {
+                Self.attemptCount = 1
```

Comment:
should this be initially 0 or 1?

Reply:
it should be 1 as whenever `initializeLivenessStream()` is called - either that attempt is the first attempt for new interval or adds to previous attempt count